### PR TITLE
data: saram7619

### PR DESCRIPTION
**Pull request by saram7619**

### CODE REVIEW

**Review by sital002 — CHANGES_REQUESTED** [review]
The file name must match your github username.

**Review by sital002 — APPROVED** [review]

File: data/saram7619.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+"name": "Sara McClelland",
+"bio": "I am a Software Engineer, a stay at home parent and dog owner. I like coffee, reading and building ideas with coding.",
+"tags": ["open source"]
```
Comment (SaraJaoude) [inline data/saram7619.json:4]: add a comma:
```suggestion
"tags": ["open source"],
```

File: data/saram7619.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  {
+    "name": "Github",
+    "url": "https://github.com/saram7619",
+    "icon": "FaGithub"
+  },
+  {
+    "name": "Leetcode",
+    "url": "https://leetcode.com/saram7619/",
+    "icon": "SiLeetcode"
+  },
```
Comment (SaraJaoude) [inline data/saram7619.json:25]: remove comma:
```suggestion
  }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**